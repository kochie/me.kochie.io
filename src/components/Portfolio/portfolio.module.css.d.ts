export const portfolio: string;
export const portfolioText: string;
export const portfolioItem: string;
export const portfolioImage: string;
export const portfolioImageWrapper: string;
