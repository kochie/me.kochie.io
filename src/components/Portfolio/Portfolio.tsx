import * as React from "react";

import "./Portfolio.scss";

import Space from "./images/space.png";
import Particlez from "./images/particlez.png";
import Guardian from "./images/guardian.png"
import Tensorflow from "./images/tensorflow.jpeg"
import Juice from "./images/juice.png"
import Keyboard from "./images/keyboard.png"
import { createHash } from "crypto";

const projects = [
  {
    image: Space,
    title: "Space",
    description: "A small webapp that follows the mouse.",
    link: "https://kochie.space"
  },
  {
    image: Particlez,
    title: "Particlez",
    description: "A real time n-body particle simulator for the web.",
    link: "https://3body.now.sh"
  },
  {
    image: Guardian,
    title: "Guardian",
    description: "An experiment in transferring cyptographic keys between devices without the user needing to copy keys themselves.",
    link: "https://github.com/kochie/guardian-server"
  },
  {
    image: Tensorflow,
    title: "Optimising Neural Networks for Embedded Systems",
    description: "My final year project - focusing on modifying the activation layers of convolutional neural networks to work better on embedded devices.",
    link: "https://bitbucket.org/kochie/fyp"
  },
  {
    image: Juice,
    title: "Juice",
    description: "A Json-Web-Token analyser.",
    link: "https://github.com/kochie/juice"
  },
  {
    image: Keyboard,
    title: "McGurrin",
    description: "A tool to help with learning to touch type.",
    link: "https://github.com/kochie/mcgurrin"
  }
];

export default class Portfolio extends React.Component {
  render() {
    return (
      <div>
        <h2>Recent Work</h2>
        <div className="portfolio">
          {projects.map(project => (
            <div className="portfolio-item" key={createHash('md5').update(project.link).digest("base64")}>
              <figure className="portfolio-image-wrapper">
                <a href={project.link}>
                  <img src={project.image} className="portfolio-image" />
                  {/* <span className="portfolio-text">View</span> */}
                </a>
              </figure>
              <h3>{project.title}</h3>
              <p>{project.description}</p>
            </div>
          ))}
        </div>
      </div>
    );
  }
}
