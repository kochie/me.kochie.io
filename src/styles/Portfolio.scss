@import './constants';

.portfolio {
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-template-rows: auto;
  grid-gap: 5%;
  @media only screen and (max-width: $mobile-width) and (max-height: $mobile-height) and (min-resolution: $mobile-resolution),
    only screen and (max-width: $mobile-height) and (max-height: $mobile-width) and (min-resolution: $mobile-resolution) {
    grid-template-columns: 1fr;
  }
}

.portfolio-image {
  width: 100%;
  border-radius: 0.35em;
  opacity: 0.5;
  transition: opacity 0.2s ease-in-out;
  @media only screen and (max-width: $mobile-width) and (max-height: $mobile-height) and (min-resolution: $mobile-resolution),
  only screen and (max-width: $mobile-height) and (max-height: $mobile-width) and (min-resolution: $mobile-resolution) {
    opacity: 1;
  }
}

.portfolio-image-wrapper {
  background-color: black;
  border-radius: 0.35em;
}

.portfolio-image:hover {
  opacity: 1;
  transition: opacity 0.2s ease-in-out;
}

.portfolio-item {
  place-self: center;
  @media only screen and (max-width: $mobile-width) and (max-height: $mobile-height) and (min-resolution: $mobile-resolution),
  only screen and (max-width: $mobile-height) and (max-height: $mobile-width) and (min-resolution: $mobile-resolution) {
    text-align: center;
  }
}

