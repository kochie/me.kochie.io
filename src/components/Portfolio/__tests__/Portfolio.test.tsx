import * as renderer from 'react-test-renderer'
import * as React from 'react'

import Portfolio from '../Portfolio'

it('portfolio renders correctly', () => {
  const tree = renderer.create(<Portfolio />).toJSON()
  expect(tree).toMatchSnapshot()
})
