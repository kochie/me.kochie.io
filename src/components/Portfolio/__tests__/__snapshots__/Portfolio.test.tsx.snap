// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`portfolio renders correctly 1`] = `
<div>
  <h2>
    Recent Work
  </h2>
  <div
    className="grid 2xl:grid-cols-3 md:grid-cols-2 grid-cols-1 grid-flow-row gap-10 auto-cols-max"
  >
    <div
      className="group rounded-xl bg-cool-gray-300 dark:bg-cool-gray-700 hover:shadow-2xl shadow-xl transition duration-300"
    >
      <div
        className="relative rounded-t-xl overflow-hidden"
        style={
          Object {
            "padding": "30% 0",
          }
        }
      >
        <a
          href="https://kochie.space"
        >
          <div
            style={
              Object {
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "left": 0,
                "margin": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <noscript>
              <img
                alt="Space"
                className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
                data-nimg={true}
                decoding="async"
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Fspace.webp&w=1200&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fspace.webp&w=320&q=75 320w, /_next/image?url=%2Fimages%2Fspace.webp&w=420&q=75 420w, /_next/image?url=%2Fimages%2Fspace.webp&w=768&q=75 768w, /_next/image?url=%2Fimages%2Fspace.webp&w=1024&q=75 1024w, /_next/image?url=%2Fimages%2Fspace.webp&w=1200&q=75 1200w"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
            <img
              alt="Space"
              className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
              data-nimg={true}
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
      <div
        className="px-5"
      >
        <h3
          className="text-lg mt-0"
        >
          <a
            href="https://kochie.space"
          >
            Space
          </a>
        </h3>
        <p
          className="text-sm"
        >
          A small webapp that follows the mouse.
        </p>
      </div>
    </div>
    <div
      className="group rounded-xl bg-cool-gray-300 dark:bg-cool-gray-700 hover:shadow-2xl shadow-xl transition duration-300"
    >
      <div
        className="relative rounded-t-xl overflow-hidden"
        style={
          Object {
            "padding": "30% 0",
          }
        }
      >
        <a
          href="https://3body.now.sh"
        >
          <div
            style={
              Object {
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "left": 0,
                "margin": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <noscript>
              <img
                alt="Particlez"
                className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
                data-nimg={true}
                decoding="async"
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Fparticlez.webp&w=1200&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fparticlez.webp&w=320&q=75 320w, /_next/image?url=%2Fimages%2Fparticlez.webp&w=420&q=75 420w, /_next/image?url=%2Fimages%2Fparticlez.webp&w=768&q=75 768w, /_next/image?url=%2Fimages%2Fparticlez.webp&w=1024&q=75 1024w, /_next/image?url=%2Fimages%2Fparticlez.webp&w=1200&q=75 1200w"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
            <img
              alt="Particlez"
              className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
              data-nimg={true}
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
      <div
        className="px-5"
      >
        <h3
          className="text-lg mt-0"
        >
          <a
            href="https://3body.now.sh"
          >
            Particlez
          </a>
        </h3>
        <p
          className="text-sm"
        >
          A real time n-body particle simulator for the web.
        </p>
      </div>
    </div>
    <div
      className="group rounded-xl bg-cool-gray-300 dark:bg-cool-gray-700 hover:shadow-2xl shadow-xl transition duration-300"
    >
      <div
        className="relative rounded-t-xl overflow-hidden"
        style={
          Object {
            "padding": "30% 0",
          }
        }
      >
        <a
          href="https://github.com/kochie/guardian-server"
        >
          <div
            style={
              Object {
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "left": 0,
                "margin": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <noscript>
              <img
                alt="Guardian"
                className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
                data-nimg={true}
                decoding="async"
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Fguardian.webp&w=1200&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fguardian.webp&w=320&q=75 320w, /_next/image?url=%2Fimages%2Fguardian.webp&w=420&q=75 420w, /_next/image?url=%2Fimages%2Fguardian.webp&w=768&q=75 768w, /_next/image?url=%2Fimages%2Fguardian.webp&w=1024&q=75 1024w, /_next/image?url=%2Fimages%2Fguardian.webp&w=1200&q=75 1200w"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
            <img
              alt="Guardian"
              className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
              data-nimg={true}
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
      <div
        className="px-5"
      >
        <h3
          className="text-lg mt-0"
        >
          <a
            href="https://github.com/kochie/guardian-server"
          >
            Guardian
          </a>
        </h3>
        <p
          className="text-sm"
        >
          An experiment in transferring cyptographic keys between devices without the user needing to copy keys themselves.
        </p>
      </div>
    </div>
    <div
      className="group rounded-xl bg-cool-gray-300 dark:bg-cool-gray-700 hover:shadow-2xl shadow-xl transition duration-300"
    >
      <div
        className="relative rounded-t-xl overflow-hidden"
        style={
          Object {
            "padding": "30% 0",
          }
        }
      >
        <a
          href="https://bitbucket.org/kochie/fyp"
        >
          <div
            style={
              Object {
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "left": 0,
                "margin": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <noscript>
              <img
                alt="Optimising Neural Networks for Embedded Systems"
                className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
                data-nimg={true}
                decoding="async"
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Ftensorflow.webp&w=1200&q=75"
                srcSet="/_next/image?url=%2Fimages%2Ftensorflow.webp&w=320&q=75 320w, /_next/image?url=%2Fimages%2Ftensorflow.webp&w=420&q=75 420w, /_next/image?url=%2Fimages%2Ftensorflow.webp&w=768&q=75 768w, /_next/image?url=%2Fimages%2Ftensorflow.webp&w=1024&q=75 1024w, /_next/image?url=%2Fimages%2Ftensorflow.webp&w=1200&q=75 1200w"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
            <img
              alt="Optimising Neural Networks for Embedded Systems"
              className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
              data-nimg={true}
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
      <div
        className="px-5"
      >
        <h3
          className="text-lg mt-0"
        >
          <a
            href="https://bitbucket.org/kochie/fyp"
          >
            Optimising Neural Networks for Embedded Systems
          </a>
        </h3>
        <p
          className="text-sm"
        >
          My final year project - focusing on modifying the activation layers of convolutional neural networks to work better on embedded devices.
        </p>
      </div>
    </div>
    <div
      className="group rounded-xl bg-cool-gray-300 dark:bg-cool-gray-700 hover:shadow-2xl shadow-xl transition duration-300"
    >
      <div
        className="relative rounded-t-xl overflow-hidden"
        style={
          Object {
            "padding": "30% 0",
          }
        }
      >
        <a
          href="https://github.com/kochie/juice"
        >
          <div
            style={
              Object {
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "left": 0,
                "margin": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <noscript>
              <img
                alt="Juice"
                className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
                data-nimg={true}
                decoding="async"
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Fjuice.webp&w=1200&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fjuice.webp&w=320&q=75 320w, /_next/image?url=%2Fimages%2Fjuice.webp&w=420&q=75 420w, /_next/image?url=%2Fimages%2Fjuice.webp&w=768&q=75 768w, /_next/image?url=%2Fimages%2Fjuice.webp&w=1024&q=75 1024w, /_next/image?url=%2Fimages%2Fjuice.webp&w=1200&q=75 1200w"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
            <img
              alt="Juice"
              className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
              data-nimg={true}
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
      <div
        className="px-5"
      >
        <h3
          className="text-lg mt-0"
        >
          <a
            href="https://github.com/kochie/juice"
          >
            Juice
          </a>
        </h3>
        <p
          className="text-sm"
        >
          A Json-Web-Token analyser.
        </p>
      </div>
    </div>
    <div
      className="group rounded-xl bg-cool-gray-300 dark:bg-cool-gray-700 hover:shadow-2xl shadow-xl transition duration-300"
    >
      <div
        className="relative rounded-t-xl overflow-hidden"
        style={
          Object {
            "padding": "30% 0",
          }
        }
      >
        <a
          href="https://github.com/kochie/mcgurrin"
        >
          <div
            style={
              Object {
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "left": 0,
                "margin": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <noscript>
              <img
                alt="McGurrin"
                className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
                data-nimg={true}
                decoding="async"
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Fkeyboard.webp&w=1200&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fkeyboard.webp&w=320&q=75 320w, /_next/image?url=%2Fimages%2Fkeyboard.webp&w=420&q=75 420w, /_next/image?url=%2Fimages%2Fkeyboard.webp&w=768&q=75 768w, /_next/image?url=%2Fimages%2Fkeyboard.webp&w=1024&q=75 1024w, /_next/image?url=%2Fimages%2Fkeyboard.webp&w=1200&q=75 1200w"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
            <img
              alt="McGurrin"
              className="absolute top-0 left-0 w-full h-full group-hover:scale-110 transform-gpu transition duration-100 filter grayscale-35 group-hover:grayscale-0"
              data-nimg={true}
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
      <div
        className="px-5"
      >
        <h3
          className="text-lg mt-0"
        >
          <a
            href="https://github.com/kochie/mcgurrin"
          >
            McGurrin
          </a>
        </h3>
        <p
          className="text-sm"
        >
          A tool to help with learning to touch type.
        </p>
      </div>
    </div>
  </div>
</div>
`;
